.github/workflows/ci.yml

```diff
@@ -37,8 +36,8 @@ jobs:
         run: tox
 
       - name: Run coverage
-        if: matrix.python-version == '3.8' && matrix.toxenv == 'py38'
-        uses: codecov/codecov-action@v4
+        if: (matrix.python-version == '3.11' && matrix.toxenv == 'py311') || (matrix.python-version == '3.12' && matrix.toxenv == 'py312')
```

Comment:
Run coverage on only the latest python version we're supporting (3.12 in this case).